watchdog.js

```diff
@@ -165,75 +152,71 @@ return;
        var msg_text = 'KDA node sync freez detected!';
        await send_telegram_msg(emoji_title,info_type,field_type,msg_text,label);*/
 
-       sleep.sleep(3);
+      shell.exec("sleep 3");
```

Comment:
I am not sure If I am a big fun of thos change.
I think I prefered old sleep lib, instead of calling shell to sleep :)